governance.md

```diff
@@ -106,17 +106,19 @@ Membership in the Core Team is by invitation of the existing members. A nominati
 Core Team members remain on the team until they choose to retire or until 2/3 of the members vote to remove them.
 
 The current composition of the Core Team is
+
 * Charlie Poole
 * Rob Prouse
 * Chris Maddock
 * Terje Sandstrom
 * Joseph Musser
+* Mikkel Bundgaard
 
 ### Core Team Chair
 
-The Core Team Chair is a single individual, voted for by the Core Team members. Once someone has been appointed Chair, they remain in that role until they choose to retire, or the Core Team casts a two-thirds majority vote to remove them.
+The Core Team Chair is a single individual, voted for by the Core Team members. The Chair leads the team, facilitates the decision-making process and ensures that all governance processes are adhered to.
 
-The Chair has no additional authority over other members of the Core Team: the role is that of a coordinator and facilitator. The Chair is also expected to ensure that all governance processes are adhered to, and has the deciding vote in case of ties. 
+Once someone has been appointed Chair, they remain in that role until they choose to resign or the Core Team casts a two-thirds majority vote to remove them.
```

Comment:
Two changes here. 

1. I took out "The Chair has no additional authority.." because that seemed to add nothing. Happy to put it back if you feel it's needed.

2. I removed the Chair as a tie-breaker because I think that's left over from an earlier draft. It seems to suggest that the Chair doesn't vote at all, except in case of a tie. The more detailed decision.md document states clearly that the Chair votes along with everyone else. If anyone interprets this differently, let's discuss.